cli/utils/functions.sh

```diff
@@ -16,7 +16,7 @@ Commands:
     status                             Check the status of all services
     start                              Start the Citadel service
     stop                               Stop the Citadel service safely
-    restart                            Restart the Citadel service
+    restart <service>                  Restart Citadel or individual services
```

Comment:
```suggestion
    restart [<service>]                  Restart Citadel or individual services
```

Not sure what the conventional way of specifying that a parameter is optional is, but I think we should do it somehow.

Reply:
I looked at some man pages but can't really make out a clear convention. But it looks like optional args are always wrapped in just square brackets.